### PR TITLE
Send Billing Address for Universal Gateways

### PR DESCRIPTION
**Pull request by ThereExistsX**
Allow universal gateways to send their billing address as well as their shipping address.

**Attn:**
@aprofeit @andrewpaliga @girasquid

### CODE REVIEW

File: test/unit/integrations/universal/universal_helper_test.rb [hunk 1]
```diff
@@ -112,6 +112,30 @@ def test_shipping_address_fields
     assert_field 'x_customer_shipping_phone',      '(416) 123-4567'
   end
 
+  def test_billing_address_fields
+    @helper.billing_address :first_name => 'John',
+                             :last_name  => 'Doe',
```
Comment (aprofeit) [inline test/unit/integrations/universal/universal_helper_test.rb]: Stylewise, space is off here.

**Comment by aprofeit** [comment]
What does this actually change? Is this now included in the request we send to the offsite payment gateway?

**Comment by aprofeit** [comment]
If I test this out in the universal gateway simulator, will these new fields show up?

**Comment by ThereExistsX** [comment]
@aprofeit : Yep, you can pull down the branch and test it using any universal gateway you have including the Bogus Gateway.

File: test/unit/integrations/universal/universal_helper_test.rb (hunk 1 above)
Comment (ThereExistsX) [inline test/unit/integrations/universal/universal_helper_test.rb]: @aprofeit : Should be fixed now.

**Comment by aprofeit** [comment]
I think I may be missing something, how does this affect the bogus gateway?

**Comment by aprofeit** [comment]
Looks like everything shows up correctly in offsite simulator
screen shot 2016-02-04 at 1 15 06 pm
:+1:

**Comment by andrewpaliga** [comment]
Nice job -- we'll need to notify the gateways before merging this change though.

**Comment by aprofeit** [comment]
@andrewpaliga 
> Nice job -- we'll need to notify the gateways before merging this change though.

Do we really? These are just additional fields we're sending them, shouldn't affect them, right?

**Comment by andrewpaliga** [comment]
@aprofeit in theory yes, but good practice to give them a heads up just in case no? ie. they might have a check for number of fields or something silly

**Comment by ThereExistsX** [comment]
@andrewpaliga -- Just let me know when this is good to ship!

**Comment by aprofeit** [comment]
> they might have a check for number of fields or something silly

I guess it's possible, but would be pretty weird. Your call.

**Comment by captn3m0** [comment]
It doesn't break our code (@Razorpay), but I'd still have liked a heads-up for this change.

**Pull request closed without merging**